lightly/models/utils.py

```diff
@@ -638,13 +639,15 @@ def add_stochastic_depth_to_blocks(vit: Module, prob: float = 0.0, mode="row") -
             mod.mlp = Sequential(mod.mlp, StochasticDepth(p=prob, mode=mode))
 
 
-def initialize_2d_sine_cosine_positional_embedding(pos_embedding: Parameter) -> None:
+def initialize_2d_sine_cosine_positional_embedding(
+    pos_embedding: Parameter, has_class_token: bool
```

Comment:
Could you make `has_class_token` optional? In the other parts of the code where we use this we do not pass a value for `has_class_token`

Reply:
I updated the other parts of the code. I think it is better to not have it optional because it is easy to miss.

Reply:
ok! If you are sure that you have updated all affected parts, feel free to resolve.